src/appointment/appointment.service.ts

```diff
@@ -0,0 +1,94 @@
+import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
+import { InjectModel } from 'nestjs-typegoose';
+import { Appointment } from 'src/model/appointment.model';
+import {AppointmentStatus} from 'src/enum/appointment.enum'
```

Comment:
Add `whitespace` after `{` and before `}`. It should look like this `import { AppointmentStatus } from ...`